src/evergreen/cli/main.py

```diff
@@ -75,6 +75,24 @@ def list_projects(ctx):
     click.echo(fmt_output(fmt, projects))
 
 
+@cli.command()
+@click.pass_context
+@click.option('--project', required=True)
+@click.option('--limit', type=int)
+def list_versions(ctx, project, limit):
+    api = ctx.obj['api']
+    fmt = ctx.obj['format']
+    version_list = api.versions_by_project(project)
+    versions_to_display = []
+    for i, version in enumerate(version_list):
+        if i > limit:
+            break
+
+        versions_to_display.append(version)
+
```

Comment:
What do you think of `itertools.takewhile(lambda i, version: i <= limit, enumerate(version_list))` for this?

Reply:
updated.